website/content/docs/nia/enterprise/license.mdx

```diff
@@ -21,19 +22,25 @@ To get a trial license for Consul-Terraform-Sync Enterprise, you can sign-up for
 Choose one of the following methods (in order of precedence) to set the license:
 
 1. Set the `CONSUL_LICENSE` environment variable to the license string.
- ```shell-session
- export CONSUL_LICENSE=<LICENSE_VALUE>
- ```
+
+```shell-session
+export CONSUL_LICENSE=<LICENSE_VALUE>
+```
+
 2. Set the `CONSUL_LICENSE_PATH` environment variable to the path of the file containing the license.
- ```shell-session
- export CONSUL_LICENSE_PATH=<PATH>/<TO>/<FILE>
- ```
+
+```shell-session
+export CONSUL_LICENSE_PATH=<PATH>/<TO>/<FILE>
+```
+
 3. Configure the [`license_path`](/docs/nia/configuration#license_path) option in the configuration file to point to the file containing the license.
- ```hcl
- license_path = "<PATH>/<TO>/<FILE>"
- ```
+
+```hcl
+license_path = "<PATH>/<TO>/<FILE>"
+```
```

Comment:
Hi! Thanks for improving our CTS docs! Apologies to comment after merge.

I previously worked on this license option formatting. I actually had a hard time getting the bullet numbering to be accurate and thought indenting sample shell/hcl would clarify it was a child of the licensing option description.

With this formatting update, it looks like the bullet numbering renders as all 1s and the sample shell/hcl isn't indented (I'm ok with no indentation and good to default to your opinion)

<img width="947" alt="Screen Shot 2021-09-02 at 10 31 50 AM" src="https://user-images.githubusercontent.com/6819378/131862922-6bf3bf6c-73c8-4036-93f7-c76a93c736d5.png">